packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -933,6 +940,12 @@ class NavigationRailDestination {
   ///    icons.
   final Widget selectedIcon;
 
+  /// The color of the [indicatorShape] when this destination is selected.
+  final Color? indicatorColor;
```

Comment:
The indicatorColor and indicatorShape API doc should explain how they're related to NavigationRail's properties and likewise for NavigationRailTheme.